hermes-api/src/main/java/pl/allegro/tech/hermes/api/constraints/EndpointAddressValidator.java

```diff
@@ -44,9 +55,16 @@ private boolean validateWithTemplate(String address, ConstraintValidatorContext
         return true;
     }
 
-    protected void createConstraintMessage(ConstraintValidatorContext context, String message) {
+    private void createConstraintMessage(ConstraintValidatorContext context, String message) {
         context.buildConstraintViolationWithTemplate(message)
                 .addConstraintViolation()
                 .disableDefaultConstraintViolation();
     }
+
+    private boolean isInvalidHost(UriTemplate template) {
+        Map<String, Object> uriKeysWithEmptyValues = Arrays.asList(template.getVariables()).stream().collect(toMap(v -> v, v -> "empty"));
```

Comment:
You can use Function.identity() to express function that returns input arguments.